src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -350,10 +353,11 @@ export const Checkout = ({
     quoteCartId,
     storage,
     variation,
+    isCompletingCheckout,
   ])
 
   useEffect(() => {
-    if (checkoutStatus === CheckoutStatus.Signed && !isCompletingCheckout) {
+    if (checkoutStatus === CheckoutStatus.Signed) {
       completeCheckout()
     }
   }, [checkoutStatus, completeCheckout, isCompletingCheckout])
```

Comment:
We probably can remove `isCompletingCheckout` from de dependency array, right?!

Reply:
Sloppy of me :P